webapp/src/components/ProgramCalendar.tsx

```diff
@@ -69,6 +69,26 @@ const ProgramCalendar = ({ program }: ProgramCalendarProps) => {
         minHeight: '4%',
       };
     }
+    if (event.allDay && currentView !== 'agenda') {
+      style = {
+        backgroundColor: 'midnightblue',
+      };
+    }
+
+    if (event.allDay && currentView === 'week') {
+      style = {
+        height: '190%',
+        backgroundColor: 'midnightblue',
+      };
+    }
+
+    if (event.allDay && currentView === 'day') {
+      style = {
+        height: '220%',
+        backgroundColor: 'midnightblue',
```

Comment:
I noticed that we are repeating code here `backgroundColor: 'midnightblue'` but it seems like it's a little bit tricky to work with because of the type of style.
Otherwise everything works as described and running yarn delint didn't change any formatting.